kolibri/core/auth/api.py

```diff
@@ -269,15 +269,18 @@ class FacilityUsernameViewSet(ValuesViewset):
     values = ("username",)
 
     def get_queryset(self):
-        if valid_app_key_on_request(self.request):
-            # Special case for app context to return usernames for
-            # the list display
-            return FacilityUser.objects.all()
```

Comment:
This code branch is required and shouldn't be deleted.

Reply:
Please review again, if still not up to the mark please tell what I am doing wrong.